OrbitGl/Batcher.cpp

```diff
@@ -103,6 +103,25 @@ void Batcher::AddTriangle(const Triangle& triangle, const Color& color, const Co
   user_data_.push_back(std::move(user_data));
 }
 
+void Batcher::AddCircle(Vec2 position, float radius, float z, Color color) {
+  std::vector<Vec2> circle_points_scaled_by_radius;
+  for (auto& point : circle_points) {
+    circle_points_scaled_by_radius.emplace_back(radius * point);
+  }
+
+  Vec2 prev_point(position[0], position[1] - radius);
```

Comment:
nit. Oh I guess we could simplify further by making this a Vec3 using the "z" component here. Feel free to leave as is.

Reply:
I will update the code! Thanks a lot!